app/index.html

```diff
@@ -469,23 +476,23 @@ <h3 class="userDetails-filename profileUserName"></h3>
             </div>
         </section>
 
-        <section id="popupDelete" class="popup popup_delete is-open j-popupDelete">
+        <section id="popupDeleteContact" class="popup popup_delete is-open j-popupDeleteContact">
```

Comment:
check out what the semantic block you have to.
I think it's not a section, maybe it's a aside.